certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+              "required": [
+                "published"
+              ]
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/envelopes/{envelope-id}/documents/{document-id}/templates/{template-id}/user-data": {
+      "post": {
+        "operationId": "PostEnvelopesEnvelopeIdDocumentsDocumentIdTemplatesTemplateIdUserData",
+        "summary": "Generate Document from Template",
+        "description": "Generates a document from a preconfigured template.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/DocumentId"
+          },
+          {
+            "name": "template-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Template Id",
+            "description": "The Id of the template which should be used",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object"
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/abort-status": {
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeIdAbortStatus",
+        "summary": "Set envelope abort status",
+        "description": "Sets the status of a given envelope to aborted",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "comment": {
+                  "type": "string"
+                }
+              },
+              "x-ms-visibility": "advanced"
+            },
+            "required": false,
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-abort